yarn.lock

```diff
@@ -6542,6 +6542,10 @@ isurl@^1.0.0-alpha5:
     has-to-string-tag-x "^1.2.0"
     is-object "^1.0.1"
 
+"jellyfin-apiclient@https://github.com/MrTimscampi/jellyfin-apiclient-javascript#jf-web-part2":
```

Comment:
Same here

Reply:
This will be changed automatically once the one in package.json is taken care of.